### PR TITLE
Update helper functions to the latest version

### PR DESCRIPTION
**Pull request by potocnyj**
Update the create-stat-functions scripts to include the latest necessary changes for Postgres monitoring.

### CODE REVIEW

**Comment by potocnyj** [comment]
@efepapa PTAL, I plan on rolling out the agents next week so no rush on this 👍

File: static/create-stat-functions-v84.sql [hunk 1]
```diff
@@ -36,24 +36,24 @@ $$ LANGUAGE sql VOLATILE SECURITY DEFINER;
...
+  c14 numeric, c15 double precision, c16 double precision, c17 oid) AS
 $$
   SELECT MD5(query), SUM(calls), 1000 * SUM(total_time), SUM(rows),
       0::numeric, 0::numeric, 0::numeric, 0::numeric, 0::numeric, 0::numeric, 0::numeric,
-      0::numeric, 0::numeric, 0::numeric, 0::double precision, 0::double precision
-      FROM pg_stat_statements WHERE query <> '<insufficient privilege>'
-      GROUP BY MD5(query);
+      0::numeric, 0::numeric, 0::numeric, 0::double precision, 0::double precision, dbid
+      FROM pg_stat_statements
+      GROUP BY MD5(query), dbid;
```
Comment (efepapa) [inline static/create-stat-functions-v84.sql]: The query we use in the superuser case groups by `dbid, MD5(query)`.  Please switch the order in one of them.

Comment (potocnyj) [inline static/create-stat-functions-v84.sql]: I'll switch the agent version, IMO it's clearer to have the columns grouped in the same order we specify in the select.

**Comment by potocnyj** [comment]
We released the latest version of agents yesterday which includes support for the new helper functions, so I'm going to go ahead and merge this now.

**Pull request merged**